cmd/ip-masq-agent/ip-masq-agent.go

```diff
@@ -318,6 +318,7 @@ func (m *MasqDaemon) syncMasqRules() error {
 	writeMasqRules(lines, toPorts)
 
 	writeLine(lines, "COMMIT")
+	klog.V(2).Infof("IPv4 masquerading rules: %q", lines)
```

Comment:
This seems to log on every periodic run (every 60s)? Let's try to avoid logging when unnecessary - this ultimately may increase cost on the user (especially in cloud provider environments).

Could we rate limit this and also log if rule changes?

Reply:
done.